### ISSUE TITLE
TF not functioning on certain app tray menus 

### ISSUE THREAD

**Issue by Undisputed00x** [ALTaleX531/TranslucentFlyouts#131]
Some app tray menus are not getting affected by TFv3 (latest release) at all, both using the x64 and x32 versions and still no result. 
This issue is from V2 (i dont know about V1) and i thought because no one has made a ticked about it i would make one.
An interesting thing is that these tray menus are getting properly modified by my custom windows theme which raises more questions about the problem with TF.
Win11 23H2
Using apps that maybe can conflict: RTB, TTB, MFE, DWMBG, SmartTaskbar
My TF registry settings: 
[TFreg.zip](https://github.com/ALTaleX531/TranslucentFlyouts/files/14829342/TFreg.zip)

Below some examples
<details><summary>Details</summary>
<p>

Nvidia tray menu:
![Screenshot 2024-04-02 014910](https://github.com/ALTaleX531/TranslucentFlyouts/assets/80290008/9c16017a-c3e8-4085-a1ae-2ed87fda7d3b)

MsiAfterburner tray menu:
![Screenshot 2024-04-02 015146](https://github.com/ALTaleX531/TranslucentFlyouts/assets/80290008/b23716a7-ca11-409a-b697-13c63b730a19)
SmartTaskbar tray menu:
![Screenshot 2024-04-02 015453](https://github.com/ALTaleX531/TranslucentFlyouts/assets/80290008/d98917bc-4b08-46da-a18e-e732963e35a3)

Functioning examples:
MemReduct:
![Screenshot 2024-04-02 015310](https://github.com/ALTaleX531/TranslucentFlyouts/assets/80290008/39e17e05-66df-42b8-8f2a-7c2c20b31639)
Windows Security Center:
![Screenshot 2024-04-02 020341](https://github.com/ALTaleX531/TranslucentFlyouts/assets/80290008/c7f71dfb-e12c-4daf-9af1-75c09b0e7c65)
</p>
</details>

**Comment by ALTaleX531** [comment]
This is probably caused by the change to uxtheme on 23H2, the other unaffected ones are called immersive context menus. I will check this issue later.

**Comment by Undisputed00x** [comment]
Any update on this issue...

**Comment by ALTaleX531** [comment]
@Undisputed00x 
Let me simply explain why TF doesn't work on the tray menus of some applications, firstly for NVIDIA Control Panel, it seems that TF doesn't have enough permissions to inject into it's process, so it doesn't work. Secondly for MsiAfterburner, its process is x86 architecture and only TF for x86 architecture can inject into it. SmartTaskbar handles some of the drawing itself, which makes TF think that it has done all the drawing by itself and should not be intervened, so TF doesn't work either.
In short, they cannot be fixed.
